src/routes/_utils/blurhash.js

```diff
@@ -5,6 +5,7 @@ import { QuickLRU } from '../_thirdparty/quick-lru/quick-lru'
 
 // A timeline will typically show 20-30 articles at once in the virtual list. The maximum number
 // of sensitive images per article is 4. 30*4=120, so this is a very conservative number.
+// Blurhash blobs seem to range from ~1.2-2kB, so this cache could grow to about 2kB*150=300kB max.
```

Comment:
Make it can be force-evicted (and the URLs revoked) when [`memorypressure`](https://www.chromestatus.com/features/5719614029824000) is emmited?

Reply:
That seems reasonable yeah, but `memorypressure` is apparently not implemented in any browser, even Chrome: https://bugs.chromium.org/p/chromium/issues/detail?id=708752